app/views/admin/reports/show.html.haml

```diff
@@ -29,6 +29,9 @@
 - if @report.comment.present?
   = render 'admin/reports/comment', report: @report
 
+- if @report.account.local? && !@statuses.empty? && (@report.account.user.settings['web.expand_content_warnings'] || @report.account.user.settings['web.display_media'] == 'show_all')
```

Comment:
The "does this user have expand CW or show all media on" check could probably either be extracted to a view hleper we could use here, or some well named method on User (and optionally delegated from account to tighten up even more) ... `User#all_content_visible?` or something ... not sure on naming.

Reply:
Maybe, but given its a single use for now, I'd probably not add a method for it